tests/test_compile_headers_common/test_compile_headers_list.h

```diff
@@ -83,6 +83,8 @@
 #include <nn/acp/title.h>
 #include <nn/act/client_cpp.h>
 #include <nn/ffl/miidata.h>
+#include <nn/pdm/pdm_c.h>
+#include <nn/pdm/pdm_cpp.h>
 #include <nn/swkbd/swkbd_cpp.h>
 #include <nn/ac.h>
 #include <nn/acp.h>
```

Comment:
You should also include `#include <nn/pdm.h>` here, just to completely cover all the headers - even trivial ones.